src/test/java/teammates/sqlui/webapi/GetFeedbackSessionSubmittedGiverSetActionTest.java

```diff
@@ -0,0 +1,112 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.Mockito.when;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.collections.Sets;
+
+import teammates.common.datatransfer.InstructorPrivileges;
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.FeedbackSession;
+import teammates.storage.sqlentity.Instructor;
+import teammates.ui.output.FeedbackSessionSubmittedGiverSet;
+import teammates.ui.webapi.GetFeedbackSessionSubmittedGiverSetAction;
+import teammates.ui.webapi.JsonResult;
+
+/**
+ * SUT: {@link GetFeedbackSessionSubmittedGiverSetAction}.
+ */
+public class GetFeedbackSessionSubmittedGiverSetActionTest extends
+        BaseActionTest<GetFeedbackSessionSubmittedGiverSetAction> {
+
+    private Instructor instructor;
+    private Course course;
+    private FeedbackSession feedbackSession;
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.SESSION_SUBMITTED_GIVER_SET;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @BeforeMethod
+    void setUp() {
+        course = generateCourse1();
+        instructor = generateInstructor1InCourse(course);
+        feedbackSession = generateSession1InCourse(course, instructor);
+    }
+
+    @Test
+    protected void testExecute_notEnoughParams_verifyHttpParamFailure() {
+        loginAsInstructor(instructor.getGoogleId());
+        verifyHttpParameterFailure();
+    }
+
+    @Test
+    protected void testExecute_getGiverSet_success() {
+        Set<String> givers = new HashSet<>();
+        givers.addAll(Arrays.asList(new String[] {
```

Comment:
this test is checking what we have given it, so its not really testing anything. Since nothing much goes on in the action, we can just add a new test to FeedbackSessionsDbIT (as nothing much goes on in the logic layer either)

Reply:
Resolved by adding a test in the logic layer. I didn't add it to DB layer since there is no change to db methods since the extracting of giver data is done from the Logic after fetching the FeedbackSession